src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -328,7 +333,7 @@ exports.createFiberFromText = function(content : string, priorityLevel : Priorit
   return fiber;
 };
 
-function createFiberFromElementType(type : mixed, key : null | string) : Fiber {
+function createFiberFromElementType(type : mixed, key : null | string, debugOwner : null | Fiber | ReactInstance) : Fiber {
```

Comment:
I couldn’t find a better solution.

Using the "current fiber" was wrong here because the thing being created *is* the current fiber, but it's not created yet.